app/grandchallenge/evaluation/templates/evaluation/evaluation_incomplete_jobs_detail.html

```diff
@@ -31,6 +32,9 @@ <h3 class="card-title">Prerequisite Jobs</h3>
                         <td>
                             {% include "algorithms/job_status_detail.html" with object=job %}
                         </td>
+                        <td>
+                            {% include "algorithms/job_error_message.html" with object=job %}
```

Comment:
This does not need to be a separate partial and the entirety of stderr should not be presented here:

```suggestion
                            {{ object.error_message }}
```

Please also remove the separate file.

Reply:
sure, I will remove it

Reply:
Done